devtools/server/actors/root.js

```diff
@@ -14,6 +14,8 @@ const { DebuggerServer } = require("devtools/server/main");
 loader.lazyGetter(this, "ppmm", () => {
   return Cc["@mozilla.org/parentprocessmessagemanager;1"].getService(Ci.nsIMessageBroadcaster);
 });
+loader.lazyRequireGetter(this, "WindowActor",
+  "devtools/server/actors/window", true);
```

Comment:
It means it will destructure the name from the import. It's a common pattern in DevTools, so I don't think a comment is needed at each use.